CHANGELOG.md

```diff
@@ -33,6 +34,13 @@ certain VM instructions are used.
 - Fixed compilation with latest debian gcc-arm-none-eabi
 - Fix `network:stop/0` on ESP32 so the network can be started again
 - Fix a memory corruption caused by `binary:split/2,3`
+- Add missing documentation and function specs for uart driver
+- Fixed possible concurrency problems in ESP32 UART driver
+- Fixed memory under allocation when creating ESP32 UART return messages
```

Comment:
There were many issues in this driver, but I'm not sure it was missing `REF_SIZE` allocation.